snippets/osdk-deprecation.adoc

```diff
@@ -1,3 +1,5 @@
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
snippet?

Reply:
Dang it good catch. I was going back and forth between updates for snippets and assemblies. Ty!